components/opponent/commons/opponent.lua

```diff
@@ -171,7 +171,8 @@ end
 ---@param opponent standardOpponent?
 ---@return boolean
 function Opponent.isEmpty(opponent)
-	if not opponent or not opponent.type or Table.deepEquals(opponent, Opponent.blank(opponent.type)) then
+	if not opponent or not opponent.type or
+		(Table.deepEquals(opponent, Opponent.blank(opponent.type)) and not Opponent.isTbd(opponent)) then
```

Comment:
is `and not` intended here?

Reply:
yes for team opponents .blank returns a TBD one:
https://github.com/Liquipedia/Lua-Modules/blob/07bec7b535a10dc5ed50c2c6777d8f423a091311/components/opponent/commons/opponent.lua#L118
so to catch that need to exclude it
sicne TBD Opponents are not empty from my understanding